### DISCUSSION TITLE
Mono submenu

### DISCUSSION THREAD

**Discussion by hermesmail** [GetPublii/Publii#1692] [Help] [answered]
Hi, I'm testing the mono theme. I can create submenus but they do not appear. Is this a limitation of the theme? Thanks for your help!

**Comment by bobmitro** [comment, marked as answer]
The original theme by Huseyin Emanet does not have a submenu, and the one prepared for Publii also doesn't. There is no space for a submenu there.

**Reply by hermesmail** [reply]
Thank you !
